lib/manager/github-actions/extract.ts

```diff
@@ -1,30 +1,52 @@
+import * as githubTagsDatasource from '../../datasource/github-tags';
 import { logger } from '../../logger';
 import * as dockerVersioning from '../../versioning/docker';
+import * as looseVersioning from '../../versioning/loose';
 import { PackageDependency, PackageFile } from '../common';
 import { getDep } from '../dockerfile/extract';
 
 export function extractPackageFile(content: string): PackageFile | null {
   logger.debug('github-actions.extractPackageFile()');
   const deps: PackageDependency[] = [];
   for (const line of content.split('\n')) {
-    // old github actions syntax will be deprecated on September 30, 2019
-    // after that, the first line can be removed
-    const match =
-      /^\s+uses = "docker:\/\/([^"]+)"\s*$/.exec(line) ||
-      /^\s+uses: docker:\/\/([^"]+)\s*$/.exec(line);
-    if (match) {
-      const [, currentFrom] = match;
+    if (line.trim().startsWith('#')) {
+      continue; // eslint-disable-line no-continue
+    }
+
+    const dockerMatch = /^\s+uses: docker:\/\/([^"]+)\s*$/.exec(line);
+    if (dockerMatch) {
+      const [, currentFrom] = dockerMatch;
       const dep = getDep(currentFrom);
       logger.debug(
         {
           depName: dep.depName,
           currentValue: dep.currentValue,
           currentDigest: dep.currentDigest,
         },
-        'Docker image inside GitHub Actions'
+        'Docker image inside GitHub Workflow'
       );
       dep.versioning = dockerVersioning.id;
       deps.push(dep);
+      continue; // eslint-disable-line no-continue
+    }
+
+    const tagMatch = /^^\s+-?\s+?uses: (?<depName>[\w-]+\/[\w-]+)(?<path>.*)?@(?<currentValue>.+?)\s*?$/.exec(
+      line
+    );
+    if (tagMatch?.groups) {
+      const { depName, currentValue } = tagMatch.groups;
+      if (looseVersioning.api.isValid(currentValue)) {
+        const dep: PackageDependency = {
+          depName,
+          currentValue,
+          commitMessageTopic: '{{depName}}} action',
+          datasource: githubTagsDatasource.id,
+          versioning: looseVersioning.id,
+          rangeStrategy: 'pin',
```

Comment:
I'm not sure if this changes something for `loose` versioning. 🤔

Reply:
Stole this from another manager, can drop it if it does nothing here haha

Reply:
```suggestion
```

Ok, so lets remove it.